### DISCUSSION TITLE
Checked your Docs page nothing on how to use in Unraid

### DISCUSSION THREAD

**Discussion by thorrrr** [containrrr/watchtower#1704] [General]
Hello Guys
Looking at your great documents page  but i see noting in search on how to deploy on Unraid even though you have a container for it!
Do you run it manually or do i need a script or do i do it in the environment or do i need to edit a config file?
Nothing to help a new user on Unraid please could you add something or point me to where it is covered?

**Comment by piksel** [comment]
Hello, great name btw! 😁

> even though you have a container for it

What do you mean by this? I don't think there is any official integration (made by @containrrr) as far as I know. Unfortunately, I have no idea what would be necessary to run it in Unraid. 

It's not in the docs, because it's not something that we explicitly support, but if someone knows how to get it working we'd be happy to add it to the docs.

**Reply by thorrrr** [reply]
This GitHub link is from the containers project help link ???

**Reply by piksel** [reply]
> This GitHub link is

What link? Are you talking about the docker hub link at [hub.docker.com/r/containrrr/watchtower](https://hub.docker.com/r/containrrr/watchtower)?

If so, that is not strictly Unraid related and just the README from our repo.

Otherwise, any non-official distribution of `watchtower` can link here, it doesn't mean we support (or know about) them.

**Comment by thorrrr** [comment]
The container says it is yours or is this a wrong as well?
Plus other info points to you guys ????


![image](https://github.com/containrrr/watchtower/assets/8170673/3b959e45-793c-4e11-9894-ce3690a2ed83)

![image](https://github.com/containrrr/watchtower/assets/8170673/88fdd8e3-03d4-446a-9e1d-ca06297fb58e)

**Reply by piksel** [reply]
Yes, like I said, the image on dockerhub is the official one, but that is not tied to Unraid. Whatever it is you posted on the top screenshot is not something official, but some community contribution that we don't know about. It just uses our official container image. Hence why we don't have anything Unraid related in our docs.

**Comment by thorrrr** [comment]
OK i understand  now so what your saying you offer no support for it in Unraid ?

**Reply by piksel** [reply]
Yes, or at least explicitly. You are free to use it wherever you have docker, but I have not tried it on Unraid and am unaware of any special requirements to use it there.
